1365.py

```diff
@@ -4,13 +4,14 @@ def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
         ans = []
         
         for num in nums:
-            count = 0
+            idx = sorted_nums.index(num)
```

Comment:
This is still `O(N)` complexity, as I mentioned in my [previous comment](https://github.com/woke7/leetcode-python/pull/38#discussion_r606906423). I specifically said that `index()` takes `O(N)` time.